source/time_stepping/interface.cc

```diff
@@ -76,23 +86,18 @@ namespace aspect
     template <int dim>
     double
     Manager<dim>::
-    compute_time_step_size() const
+    compute_time_step_size(Reaction &reaction) const
     {
       double new_time_step = std::numeric_limits<double>::max();
-      Reaction reaction = Reaction::advance;
 
       for (const auto &plugin : active_plugins)
         {
-          std::pair<Reaction, double> answer
-            = plugin->execute();
+          double this_time = plugin->execute();
```

Comment:
I'd rename this `this_time_step` for clarity.

Reply:
see Anne's comment: rename to `this_time_step`